sdk/src/main/java/io/customer/sdk/repository/TrackRepository.kt

```diff
@@ -41,7 +46,13 @@ class TrackRepositoryImpl(
         }
 
         // if task doesn't successfully get added to the queue, it does not break the SDK's state. So, we can ignore the result of adding task to queue.
```

Comment:
This comment is no longer correct because the next line of code uses the queue status.